### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by emileighoutlaw**
Updates privacy policy language, per our re-writes and approval from Interior's Departmental Privacy Officer.

### CODE REVIEW

**Comment by OpenGlobe** [comment]
I think we can also remove (or replace) this link:

http://www.ftc.gov/bcp/conline/edcams/kidzprivacy/index.html

Appears to be broken.

File: ekip/everykid/templates/legal/privacy.html [hunk 1]
```diff
@@ -6,58 +6,29 @@
...
+<h1> Every Kid in a Park privacy policy </h1>
+
+<h2>Fourth graders</h2>
+<p>We don't collect any information that could personally identify you. 
+We do ask you to tell us the general area where you live. Knowing this helps us 
+analyze who is using the program. It also helps us make your paper pass and protect against misuse.</p>
+ 
+<p>We collect general information about our visitors so we can improve our website. 
+If you want more information, see the <a href="http://www.doi.gov/privacy.cfm">Department of the Interior Privacy Policy</a>. 
+Also see the <a href="http://www.ftc.gov/bcp/conline/edcams/kidzprivacy/index.html">Government Children's Privacy Policy </a>.</p>
```
Comment (khandelwal) [inline ekip/everykid/templates/legal/privacy.html]: This link doesn't work. So we'll have to find a new version or remove it.

**Comment by OpenGlobe** [comment]
I think I have an updated link - just a sec

**Comment by emileighoutlaw** [comment]
woot!

**Pull request merged**